pymess/backend/emails/__init__.py

```diff
@@ -53,6 +43,7 @@ def update_message(self, message, extra_sender_data=None, **kwargs):
         message.change_and_save(
             backend=fullname(self),
             extra_sender_data=extra_sender_data,
+            number_of_send_attempts=message.number_of_send_attempts + 1
```

Comment:
hlavně že si píšu helper a pak to dám do všech volání publish_message :(

Reply:
Nějak nemůžu objevit kde byla chyba :(

Reply:
koukni na názvy těch proměnných před tím a teď. Bylo tam send_attempts_count místo number_of_send_attempts